analyses/metacells/run-metacells.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This script is to run scripts in the metacell module, currently in testing
+
+# Run from the script file location
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+data_dir="../../data/current"
+project="SCPCP000001"
+sample="SCPCS000001"
+library="SCPCL000001"
+
+test_file="${data_dir}/${project}/${sample}/${library}_processed_rna.h5ad"
+results_dir="results/${project}/${sample}"
+mkdir -p ${results_dir}
+
+# run the SEACells script
+python scripts/run-seacells.py \
+    $test_file \
```

Comment:
```suggestion
    ${test_file} \
```

Reply:
Sure, but not really important here, as there is not really any ambiguity in the variable.